wrappers/javascript/anoncreds-react-native/cpp/anoncreds.cpp

```diff
@@ -129,10 +349,10 @@ jsi::Value createPresentation(jsi::Runtime &rt, jsi::Object options) {
   auto credentialsProve =
       jsiToValue<FfiList_FfiCredentialProve>(rt, options, "credentialsProve");
   auto selfAttestedNames =
-      jsiToValue<FfiStrList>(rt, options, "selfAttestedNames");
+      jsiToValue<FfiStrList>(rt, options, "selfAttestNames");
```

Comment:
I think we should change it back to `selfAttestedNames` and make the API take those keys.

Reply:
Not sure what you mean by this. I only changed the wording from "attested" to "attest" to match other classes such as the one in NodeJs and the FFI method.

Reply:
I mean that `selfAttestedNames` is grammatically correct and `selfAttestNames` is not. But if the other is used more than we should use that one for consistency.